packages/client/src/components/form/ReviewActionComponent.tsx

```diff
@@ -308,6 +309,13 @@ const ReviewActionComponent = ({
   hasErrorsOnFields
 }: IReviewActionProps & IntlShapeProps) => {
   const [showSubmitModal, setShowSubmitModal] = useState(false)
+  const { hasAnyScope, hasScope } = usePermissions()
+
+  const declarationToBeRegistered = hasScope(SCOPES.RECORD_REGISTER)
+  const declarationToBeValidated = hasAnyScope([
+    SCOPES.RECORD_SUBMIT_FOR_APPROVAL,
+    SCOPES.RECORD_SUBMIT_FOR_UPDATES
+  ])
```

Comment:
I had a long chat to @jpye-finch about this.
https://opencrvsworkspace.slack.com/archives/C07GQ39749E/p1729841794125469
That was exactly my idea, to just display the buttons based on scopes. But the existing '`actionContext` works in a hierarchy and displays text based on the highest level action you can perform, register > validate > declare. So changing that requires a redesign. So I opted to keep the hierarchy for now because those buttons are going to get moves to the action menu soon anyway .